src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  padding: 0;
+
+  font-family: Roboto, sans-serif;
+  display: flex;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  margin: 10px;
+  padding: 22px 0;
+  color: black;
+  justify-content: stretch;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.nav__link:active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 10px;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)